docs/whats-new/zowe-announcements.md

```diff
@@ -1,21 +1,40 @@
+**Featured speaker:** Adam Wolfe, OMP Ambassador and Broadcom Product Owner
 
-Client-side components are also available. The CLI is available from NPM. The version starts with 8.0.0.next - [https://www.npmjs.com/package/@zowe/cli/v/8.0.0-next.202404191414](https://www.npmjs.com/package/@zowe/cli/v/8.0.0-next.202404191414).  Explorer for VSCode is available from GitHub releases - [https://github.com/zowe/zowe-explorer-vscode/releases](https://github.com/zowe/zowe-explorer-vscode/releases). Explorer for Intellij is available from the Intellij Marketplace - [https://plugins.jetbrains.com/plugin/18688-zowe-explorer](https://plugins.jetbrains.com/plugin/18688-zowe-explorer)
+With Zowe v3 on the release horizon, **team configuration** for **Zowe CLI** and **Zowe Explorer for Visual Studio Code** becomes more relevant than ever. The Zowe Community introduced *team config* to take the burden of learning, understanding, and implementing connectivity away from every user – which is what v1 profiles required.
 
-## Zowe Version 3.0 release delay
+Join this session for a comprehensive review of team config, what it looks like, how best to leverage it, and (for those of you still using v1 profiles) best practices for converting Zowe CLI v1 profiles to team config to ensure you and your team are ready for Zowe v3!
 
-The Zowe V3 release is postponed from its original March 28 date. The schedule for <b>Zowe V2 releases will be adjusted</b> to accommodate this change.
+## New delivery date for Zowe Version 3.0
 
-We expect to have a <b>new V3 release date</b> and additional information on Zowe V2 releases by the end of March. Notifications will go out at that time.
+The Zowe V3 release NEW delivery date is **September 30, 2024**. New Zowe V3 Office Hours coming in September.
 
 For updates, see [vNext](https://www.zowe.org/vnext) in Zowe.org.
 For information about changes to Zowe components to be introduced in Zowe v3, see [Breaking changes and Important Updates in Zowe v3](breaking-changes-v3.md).
 
+## Zowe Version 3.0 technical preview
+
+Zowe publishes technical preview builds for Version 3 for all components involved.
+
+The latest versions are downloadable from the [Zowe JFrog Artifactory directory](https://zowe.jfrog.io/zowe/libs-release-local/org/zowe/nightly/v3/). The directory contains PAX and SMP/E packages. The server-side includes the **API Mediation Layer**, **Virtual Desktop**, **ZSS**, and **ZIS**. The build is released nightly.
+
+Client-side components are also available. **Zowe CLI** is available from NPM. The version starts with [8.0.0.next](https://www.npmjs.com/package/@zowe/cli/v/8.0.0-next.202404191414). **Explorer for VS Code** is available from [GitHub releases](https://github.com/zowe/zowe-explorer-vscode/releases). **Explorer for Intellij** is available from the [Intellij Marketplace](https://plugins.jetbrains.com/plugin/18688-zowe-explorer)
```

Comment:
This should be changed:
"The version starts with [8.0.0.next]"
to
"The version starts with [8.0.0-next]"